docker-compose.yml

```diff
@@ -208,6 +208,21 @@ services:
       - ./build:/app/build
       - ./test/react-native-cli/features/:/app/features/
 
+  release:
+    build:
+      context: .
+      dockerfile: dockerfiles/Dockerfile.release
+    environment:
+      GITHUB_USER:
+      GITHUB_ACCESS_TOKEN:
+      RELEASE_BRANCH:
+      AWS_ACCESS_KEY_ID:
+      AWS_SECRET_ACCESS_KEY:
+      VERSION:
+    volumes:
+      - ~/.gitconfig:/home/releaser/.gitconfig
```

Comment:
I don't think this will work for me as my `.gitconfig` looks like this:

```
[include]
    path = ~/dotfiles/.gitconfig
```

Can we use the output of `git config user.name` and `git config user.email` instead (somehow)? I'd be OK with extra environment variables for `GIT_USER`/`GIT_EMAIL` as well

Reply:
Actually it seems like Git is OK with paths that don't exist and duplicate config sections, so I can just put my user info in `~/.gitignore`

Reply:
Damn, I took a chance on that as it simplifies things a lot. It will have to be env variables, as you can't use the output of a command on the host machine as input for docker/compose.

Reply:
I tested this morning and I think it's OK as-is — I can just put my user info in `~/.gitconfig` and git is happy even when the included file doesn't exist